src/main/java/org/truffleruby/interop/InteropNodes.java

```diff
@@ -582,10 +582,6 @@ public Object read(TruffleObject receiver, Object identifier,
             return foreignToRubyNode.executeConvert(foreign);
         }
 
-        protected int getCacheLimit() {
-            return getContext().getOptions().INTEROP_READ_CACHE;
```

Comment:
It looks like we can remove this from `tool/options.yml`and regenerate `Options` & `OptionsCatalog` with `jt build options`.